project/templates/gamemakerPage.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'basic.html' %}
+{% load filters %}
+
+{% block title %} Game Maker Control Panel {% endblock %}
+{% block Nav-Bar %}
+    <a class="eScavenge">Game: {{ curgame}}</a>
+    <p>{{ message }}</p>
+    
+{% endblock %}
+{% block dropdown %}
+
+    <div class="dropdown">
+        <button class="dropbtn">Game Options</button>
+        <div class="dropdown-content">
+            <form action="/gamemakerPage/" method="post">
+                {% csrf_token %}
+                <select name = "gameObject" value="--select-">
+                    {% for g in games %}
+                    <option value={{g.name}}>{{g.name}}</option>}
+                    {% endfor %}
+                </select> 
+                <input type="submit" id="selectGame" name="selectGame" value="Select Game"></br></br>
+                <input style="background-color: rgb(56, 255, 89)" type ="submit" id="startGame" name = "startGame" value="Start Game">
+                <input style="background-color: red"type ="submit" id="endGame" name = "endGame" value="End Game"></br></br>
+                <input type ="submit" id="gmlogout" name = "gmlogout" value= "Log Out">
+            </form>
+        </div>
+    </div>
+{% endblock %}
+{% block command-box %}
+    <h3 style="color:white">Game Status:{{gamestatus}}</h3>
+    <form action="/editTeam/" method="post"></form>
```

Comment:
Why are you posting to editTeam? We have a whole other page for editing a team, you should just be redirecting there probably just with an <a> tag.